<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Choose OS</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="macBox">
          <property name="text">
           <string>Mac</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="windowsBoc">
          <property name="text">
           <string>Windows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="linuxBox">
          <property name="text">
           <string>Linux</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Choose Drink</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="coffeBox">
          <property name="text">
           <string>Coffee</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="teeBox">
          <property name="text">
           <string>Tee</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="waterBox">
          <property name="text">
           <string>Water</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Choose Answer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="aRadioButton">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bRadioButton">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cRadioButton">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dRadioButton">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="grabDataButton">
     <property name="text">
      <string>Grab Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setStateButton">
     <property name="text">
      <string>Set State</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
